server.go

```diff
@@ -9,8 +9,12 @@ import (
 	"errors"
 	"io"
 	"net"
+	"runtime"
 	"strings"
 	"sync"
+
+	"golang.org/x/sync/errgroup"
```

Comment:
This isn't formatted properly. Stdlib should be grouped up separately from other imports.